tests/tests/test_mtls.py

```diff
@@ -203,7 +168,6 @@ def common_test_mtls_enterprise(self, env, algorithm=None, use_hsm=False):
             config = json.loads(output)
             # replace mender.conf with an mTLS enabled one
             config["ServerURL"] = "https://mtls-ambassador:8080"
-            config["ServerCertificate"] = "/etc/mender/server.crt"
```

Comment:
I am having doubts here, isnt it the idea of this test to have the server certificate here? (it has been a long time, and I have not wrote this test in whole, so sorry for stupid quesitons)

Reply:
We add the certificate to `ca-certs` now :)